command/server/test-fixtures/config_custom_response_headers_multiple_listeners.hcl

```diff
@@ -56,4 +56,3 @@ listener "tcp" {
 }
 
 
-disable_mlock = true
```

Comment:
Neither here nor there, just a note mostly to myself... We should probably split a bunch of these tests to use `disable_cache=false` and make sure we exercise both paths for tests which do benefit from testing and not testing caching.